Automattic/wp-calypso
# Cart on Jetpack cloud Pricing page

#### Proposed Changes

We are enabling the purchase of multiple products on Jetpack Cloud pricing page and adding Shopping cart to achieve this. We are re-using existing Shopping Cart functionality enhancing the styles to match expected designs. 

#### Testing Instructions


* Create a new JN site
* click on Jetpack Cloud live link below
* or boot up this PR 
    * Run `git fetch && git checkout fix/cloud-cart-lightbox-cta`
    * Run `yarn start-jetpack-cloud`
    * Goto http://jetpack.cloud.localhost:3000
* Modify URL adding `/pricing/:'[JN site]?flags=jetpack/pricing-page-cart` in the end.
* Make sure the cart shows up in the header
* Test cart functionality by adding and deleting items, navigating to checkout, etc.
* Make sure Cart represented correctly on all screen sizes.

#### Pre-merge Checklist





Related to #
p6TEKc-6FI-p2
